test/controllers/admin/deploy_groups_controller_test.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../../test_helper'
+
+describe Admin::DeployGroupsController do
+  as_a_deployer do
+    unauthorized :get, :index
+    unauthorized :post, :create
+    unauthorized :delete, :destroy, id: 1
+    unauthorized :post, :update, id: 1
+    unauthorized :get, :new
+  end
+
+  as_a_admin do
+    it 'get index succeeds' do
+      get :index
+      assert_response :success
+      assigns(:deploy_groups).count.must_equal 3
+    end
+
+    unauthorized :post, :create
+    unauthorized :delete, :destroy, id: 1
+    unauthorized :post, :update, id: 1
+    unauthorized :get, :new
+  end
+
+  as_a_super_admin do
+    it 'get :index succeeds' do
+      get :index
+      assert_response :success
+      assigns(:deploy_groups).count.must_equal 3
+      deploy_groups = assigns(:deploy_groups)
+      deploy_groups.include?(deploy_groups(:deploy_group_pod1)).must_equal true
```

Comment:
not sure if this includes test is productive, there is no logic that is being tested, it just assignes all